index.html

```diff
@@ -267,7 +267,13 @@ <h3>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
-            <li>An API for Verifiable Credential Exchange</li>
+            <li>A Developer Guide consisting of one or more notes related to:
+              <ul>
+                <li>General implementation guidance and best practices for working with VCs</li>
+                <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
+                <li>Guidance on Verifiable Credential Exchange over OpenID Connect</li>
+              </ul>
+            </li>
```

Comment:
```suggestion
                <li>Verifiable Credential Exchange over GNAP</li>
                <li>Other protocols as time, attention, and resources permit.</li>
              </ul>
            </li>
```

Reply:
Let's put in a link to the GNAP specification for those in the W3C Advisory Committee that might not be aware of the work (they get cranky about acronyms that they don't have direct links for when it comes time to vote): https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol

Reply:
Done, please accept both changes when ready